apps/opik-frontend/src/components/layout/SideBar/SideBar.tsx

```diff
@@ -294,20 +283,13 @@ const SideBar: React.FunctionComponent<SideBarProps> = ({
   const { data: runningOptimizationsData } = useOptimizationsList(
     {
       workspaceName,
+      filters: ACTIVE_OPTIMIZATION_FILTER,
       page: 1,
       size: 1,
-      filters: ACTIVE_OPTIMIZATION_FILTER,
     },
     {
       placeholderData: keepPreviousData,
-      enabled: !!workspaceName && isOptimizationStudioEnabled,
-      refetchInterval: (query) => {
-        // refetch every 5 seconds if there are running optimizations
-        const data = query.state.data;
-        return data?.total && data.total > 0
-          ? RUNNING_OPTIMIZATION_REFETCH_INTERVAL
-          : false;
-      },
+      refetchInterval: RUNNING_OPTIMIZATION_REFETCH_INTERVAL,
```

Comment:
What about the !!workspaceName?

Why do we have 3 other references in code? it's confusing 😅